qemu/tests/multi_nics_stress.py

```diff
@@ -80,6 +82,15 @@ def run(test, params, env):
     client_infos = []
     server_ips = []
     client_ips = []
+
+    os_type = params.get("os_type")
+    if os_type == "windows":
+        host_mem = utils_memory.memtotal() // (1024 * 1024)
+        vm_mem = host_mem / (len(vms.split()) + 1) * 1024
```

Comment:
Hi @leidwang Would you introduce why do we need to `+1` here?

Reply:
I want to reserve some free memory. So I split the total memory into (N+1) parts